tests/NexusMods.Paths.Tests/PathExpansionTests.cs

```diff
@@ -1,15 +1,23 @@
-﻿using NexusMods.Paths.Utilities;
+﻿using NexusMods.Paths.Extensions;
+using NexusMods.Paths.Utilities;
 
 namespace NexusMods.Paths.Tests;
 
 public class PathExpansionTests
 {
+    private readonly InMemoryFileSystem _fileSystem;
+
+    public PathExpansionTests()
+    {
+        _fileSystem = new InMemoryFileSystem();
+    }
+    
     [Fact]
     public void CanExpandEntryFolder()
     {
         var inputPath = "{EntryFolder}/FluffyKitten.png";
-        var expandedPath = KnownFolders.ExpandPath(inputPath);
-        var expectedPath = Path.Combine(KnownFolders.EntryFolder.GetFullPath(), "FluffyKitten.png");
+        var expandedPath = _fileSystem.ExpandKnownFoldersPath(inputPath);
+        var expectedPath = Path.Combine(_fileSystem.GetKnownPath(KnownPath.EntryDirectory).GetFullPath(), "FluffyKitten.png");
```

Comment:
Tests like these should use `[Theory, AutoFileSystem]` using AutoFixture:

```csharp
[Theory, AutoFileSystem]
public void CanExpandEntryFolder(IFileSystem fs)

// or

[Theory, AutoFileSystem]
public void CanExpandEntryFolder(InMemoryFileSystem fs)
```

Reply:
Updated to use Theory and InlineData, good catch